src/core/execute.c

```diff
@@ -5484,6 +5484,22 @@ int exec_context_destroy_credentials(const ExecContext *c, const char *runtime_p
         return 0;
 }
 
+int exec_context_destroy_mount_ns_dir(_unused_ const ExecContext *c, const char *unit) {
```

Comment:
given context is unused, simply don't pass it?

Reply:
One reason is that I expected add more removals into this function (I thought `apply_mounts` may create various dirs as mount targets whose removal I'm not sure about -- but practically only `propagate_dir` turned out leaky so I only left that one).
Second reason is "fitting in", if I dropped the arg, the function would have no connection to ExecContext, which would obscure origin of the given resource on the cleanup path.